docs/source/contributing/developing-core.md

```diff
@@ -126,30 +139,78 @@ corepack prepare pnpm@latest --activate
 or using `npm`:
 
 ```shell
-npm install -g pnpm
+npm install -g pnpm@latest
+```
+
+Verify the latest version.
+
+```shell
+pnpm --version
 ```
 
+Compare the output to the [latest pnpm release number](https://www.npmjs.com/package/pnpm).
+
+
 ```{seealso}
 [pnpm installation](https://pnpm.io/installation).
 ```
 
 
+### Make
+
+```{include} ./install-make.md
+```
+
+
+### Docker
+
+```{include} ./install-docker.md
+```
+
+
 ## Set up the environment
 
+You need to perform the steps in this section only once to set up your environment.
+
 Clone the Volto repo, and change your working directory to the cloned repository:
 
 ```shell
 git clone https://github.com/plone/volto.git
 cd volto
 ```
 
-Install dependencies:
+Install the frontend dependencies.
 
 ```shell
 pnpm install
 ```
 
 
+## Run the environment
```

Comment:
I think this would be clearer, but I don't feel strongly:

```suggestion
## Start Volto
```

Reply:
Well, if it we only Volto, but it's also backend. "Run the environment" does not ring true either. I think I'll go with "Start the backend and Volto".